bindata/network/openshift-sdn/000-ns.yaml

```diff
@@ -9,3 +9,16 @@ metadata:
   annotations:
     openshift.io/node-selector: "" #override default node selector
     openshift.io/description: "OpenShift SDN components"
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  # NOTE: this namespace is the portal for host network traffic
+  name: openshift-host-network
+  labels:
+    openshift.io/run-level: "0"
```

Comment:
should not have that label; we don't want pods in this namespace to have any special privileges